docs/user/02-operation-guides/troubleshooting/03-50-recovering-from-unintentional-istio-removal.md

```diff
@@ -1,53 +1,80 @@
 # Istio unintentionally removed
-Follow the steps outlined in this troubleshooting guide if you unintentionally deleted Istio and want to restore the system to its normal state without losing any user-created resources. However, if you intended to delete Istio, the symptoms described in this document are expected and you must clean up the orphaned resources by yourself.
+Follow the steps outlined in this troubleshooting guide if you unintentionally deleted Istio and want to restore the system to its normal state without losing any user-created resources. However, if you intended to delete Istio, the symptoms described in this document are expected, and you must clean up the orphaned resources by yourself.
```

Comment:
```suggestion
If you unintentionally deleted Istio, follow this guide to restore the system to its normal state without losing any user-created resources. 

However, if you intended to delete Istio, the symptoms described in this document are expected, and you must clean up the orphaned resources.
```
How does the user know how to clean up which orphaned resources? Is there a guide for that? Or will readers have to figure out how to get rid of the Warning state without guidance?

Reply:
I've added information on how to see which resources are blocking deletion. We cannot offer specific guidelines as it depends on the individual use case. Since the user created these resources, we don't know how they should proceed